regolith/templates/grantreport.txt

```diff
@@ -28,17 +27,39 @@ Goal 4: Apply this to real scientific problems
 
 * What was accomplished under these goals (you must provide information for
 at least one of the 4 categories below)?
-Major Activities:
-Specific Objectives:
-Significant Results:
+------------------------------------------------------------------------------
+- Major Activities (currently worked on projecta):
```

Comment:
again, can't have me hard-coded in here.

If there is something that is "me" specific that doesn't change often it should probably be in people.  However, since this is a grant-specific thing it might better be in grants?   If it is mostly me-specific but will be described in the grant report of multiple grants, this should probably be in people but have a field that is ``grant``.  This is how we discover how the world is organized!